src/gluonts/mx/distribution/deterministic.py

```diff
@@ -88,7 +89,7 @@ def cdf(self, x):
         return is_greater_equal_or_both_nan
 
     def sample(
-        self, num_samples: Optional[int] = None, dtype=np.int32
```

Comment:
@PascalIversen was there any reason for this to be `np.int32`?

Reply:
No. Good catch, thanks!